distribution/openrct2.d.ts

```diff
@@ -71,6 +71,14 @@ declare global {
         z: number;
     }
 
+    /**
+     * A rectangular area specified using two coordinates.
+     */
+    interface MapRange {
+        leftTop: Coord2;
+        rightBottom: Coord2;
```

Comment:
Looks like a good opportunity to give Coord2 more meaningful names, it's becoming very used.